File: DBtoCSharp_MARIADB/output/INNODB_SYS_TABLESTATS.ts
export interface INNODB_SYS_TABLESTATS
    {
        /**///  */
        TABLE_ID: number;
        /**///  */
        NAME: string;
        /**///  */
        STATS_INITIALIZED: number;
        /**///  */
        NUM_ROWS: number;
        /**///  */
        CLUST_INDEX_SIZE: number;
        /**///  */
        OTHER_INDEX_SIZE: number;
        /**///  */
        MODIFIED_COUNTER: number;
        /**///  */
        AUTOINC: number;
        /**///  */
        REF_COUNT: number;
}

File: DBtoCSharp_MARIADB/modelos/INNODB_SYS_TABLESTATS.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class INNODB_SYS_TABLESTATS
    {
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long TABLE_ID { get; set; }
            [Required]
    [StringLength(64)]
        /// <summary>
        /// 
        /// </summary>
        public string NAME { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int STATS_INITIALIZED { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long NUM_ROWS { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long CLUST_INDEX_SIZE { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long OTHER_INDEX_SIZE { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long MODIFIED_COUNTER { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long AUTOINC { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public int REF_COUNT { get; set; }

}
}
